tenets/codelingo/go/declare_empty_slice/example.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+var t []string
+
+func main() {
+
+  var t []string
+  t := []string{}
+  t := [2]string{2,3}
+  
+  i := 0
+  for i < 10 {
+    var w []string
+    w := []string{}
+    w := [2]string{2,3}
```

Comment:
I don't think this will compile, but that's a minor issue - it does parse.

Reply:
which line? the whole loop?

Reply:
yep, needs to be type `int`